pxf/pxf-hbase/src/main/java/org/apache/hawq/pxf/plugins/hbase/HBaseFilterBuilder.java

```diff
@@ -71,14 +74,19 @@ public HBaseFilterBuilder(HBaseTupleDescription tupleDescription) {
      * @throws Exception if parsing failed
      */
     public Filter getFilterObject(String filterString) throws Exception {
-        FilterParser parser = new FilterParser(this);
-        Object result = parser.parse(filterString);
+        // First check for NOT, HBase does not support this
+        if (filterString.contains(NOT_OP)) {
+            return null;
+        } else {
```

Comment:
there is no point in having else branch if you return from the if branch. Removing else branch will simplify the code and make it more readable.